hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneClientMultipartUploadV1.java

```diff
@@ -566,6 +569,212 @@ public void testAbortUploadSuccessWithParts() throws Exception {
     // not making any assertion for the same.
   }
 
+  @Test
+  public void testListMultipartUploadParts() throws Exception {
+    String volumeName = UUID.randomUUID().toString();
+    String bucketName = UUID.randomUUID().toString();
+    String parentDir = "a/b/c/d/e/f";
+    String keyName = parentDir + UUID.randomUUID().toString();
+
+    store.createVolume(volumeName);
+    OzoneVolume volume = store.getVolume(volumeName);
+    volume.createBucket(bucketName);
+    OzoneBucket bucket = volume.getBucket(bucketName);
+
+    Map<Integer, String> partsMap = new TreeMap<>();
+    String uploadID = initiateMultipartUpload(bucket, keyName, STAND_ALONE,
+        ONE);
+    String partName1 = uploadPart(bucket, keyName, uploadID, 1,
+        generateData(OzoneConsts.OM_MULTIPART_MIN_SIZE, (byte)97));
+    partsMap.put(1, partName1);
```

Comment:
Can we additionally check if the partName returned is the expected value, like
``` 
Assert.assertEquals("/volumeName/bucketName....", partName1)
```

Reply:
Thanks @linyiqun for the review comments.

Yes, I'm doing partName assertion. Like we saw, it is storing `partName` in the `partsMap.put(1, partName1);` and down the line it is asserting partName against returned `ozoneMultipartUploadPartListParts`. Hope thats fine for you? Am I missing something?

    Assert.assertEquals(partsMap.get(ozoneMultipartUploadPartListParts
            .getPartInfoList().get(0).getPartNumber()),
        ozoneMultipartUploadPartListParts.getPartInfoList().get(0)
            .getPartName());
    Assert.assertEquals(partsMap.get(ozoneMultipartUploadPartListParts
            .getPartInfoList().get(1).getPartNumber()),
        ozoneMultipartUploadPartListParts.getPartInfoList().get(1)
            .getPartName());
    Assert.assertEquals(partsMap.get(ozoneMultipartUploadPartListParts
            .getPartInfoList().get(2).getPartNumber()),
        ozoneMultipartUploadPartListParts.getPartInfoList().get(2)
            .getPartName());

Reply:
@rakeshadr , I have saw above assert check . But it would be great if we could have additional assert for the value of part Name, so that we can directly know the value/format of part name.

Reply:
@linyiqun Thanks a lot for the clear explanation.
I've pushed the changes. Does this sound good to you.

Reply:
Looks good to me.